### PR TITLE
Am bdd tests

### PR DESCRIPTION
**Pull request by amiordan**

### CODE REVIEW

**Comment by CristianManic** [comment]
please separate this huge PRs in smaller ones divided by business/technical scope

**Review by CristianManic — CHANGES_REQUESTED** [review]
please separate this huge PRs in smaller ones divided by business/technical scope

File: src/test/java/cucumber_runner/TestRunner.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        plugin = { "pretty", "html:target/cucumber" }
+)
+public class TestRunner {
+
+}
+
+
+
+
+
```
Comment (CristianManic) [inline src/test/java/cucumber_runner/TestRunner.java]: remove blank spaces

File: src/test/java/hooks/TestHooks.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+public class TestHooks extends AbstractStepDef {
+
+    private ConfigFileReader fileReader;
+
+    private WebDriverManager webDriverManager;
+
+    private static void setDriver(WebDriver driver) {
+        webDriver = driver;
+    }
```
Comment (CristianManic) [inline src/test/java/hooks/TestHooks.java]: no need of this setter, you can do this in before method

File: src/test/java/hooks/TestHooks.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+    @Before
+    public void launchChromeDriver() {
+        webDriverManager = new WebDriverManager();
+        fileReader = new ConfigFileReader();
```
Comment (CristianManic) [inline src/test/java/hooks/TestHooks.java]: this action does not belongs to this method, you can do this when you initiate this

File: src/test/java/hooks/TestHooks.java [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        setDriver(webDriverManager.getDriver());
+        webDriver.navigate().to(fileReader.getApplicationUrl());
```
Comment (CristianManic) [inline src/test/java/hooks/TestHooks.java]: this should be done in first step from Background in feature file

**Review by vdones — APPROVED** [review]

**Review by MaximRaicov — APPROVED** [review]

**Pull request merged**